src/i18n/en-US/intake.ts

```diff
@@ -138,6 +137,103 @@ const intake = {
         ]
       }
     }
+  },
+  contactDetailsForm: {
+    systemDescription:
+      'The EASi System Intake process can guide you through all stages of your project, connecting you with the resources, people and services that you need. Please complete and submit this CMS IT Intake form to engage with the CMS IT Governance review process. This is the first step to receive a CMS IT LifeCycle ID. Upon submission, you will receive an email promptly from the IT_Governance mailbox, and an IT Governance Team member will reach out regarding next steps.',
+    heading: 'Contact details',
+    businessOwner: {
+      name: 'CMS Business Owner Name',
+      description:
+        'This person owns a line of business related to this request and will champion the request moving forward',
+      component: 'CMS Business Owner Component'
+    },
+    productManager: {
+      name: 'CMS Project/Product Manager, or lead',
```

Comment:
Im interested in people's thoughts here. We have a similar field in the csvHeadings section of the i18n; however, it is just "Product Manager Name". I could change the field in csvHeadings to be "CMS Project/Product Manager, or lead" but that feels weird since it would make that the name of the column in the csv file.

Reply:
This also applies to other similar fields like Business Owner Name vs CMS Business Owner Name